ios/Sources/FavoritesFeature/FavoritesScreen.swift

```diff
@@ -15,20 +15,23 @@ public struct FavoritesScreen: View {
 
     public var body: some View {
         NavigationView {
-            WithViewStore(store) { viewStore in
-                if viewStore.feedContents.isEmpty {
-                    Text("表示するコンテンツがありません")
-                } else {
-                    ScrollView {
-                        FeedContentListView(
-                            feedContents: viewStore.feedContents,
-                            tapContent: { content in
-                                viewStore.send(.tap(content))
-                            },
-                            tapFavorite: { isFavorited, contentId in
-                                viewStore.send(.tapFavorite(isFavorited: isFavorited, id: contentId))
-                            }
-                        )
+            ZStack {
+                AssetColor.Background.primary.color.ignoresSafeArea()
+                WithViewStore(store) { viewStore in
+                    if viewStore.feedContents.isEmpty {
+                        Text("表示するコンテンツがありません")
```

Comment:
In the latest `main` branch this is not used any more. We currently use [`FavoritesEmptyView`](https://github.com/DroidKaigi/conference-app-2021/blob/f31566b6c84ddbfe4f40f530f2a9f6ffeef3fbb6/ios/Sources/FavoritesFeature/FavoritesScreen.swift#L22)
I thought this should be conflicted but it's not.
Could you please fix to follow the latest `main` branch?

Reply:
It's weird to me that there is no conflict here.
I will rebase soon.